WordPress/src/main/java/org/wordpress/android/ui/jetpack/common/viewholders/JetpackButtonViewHolder.kt

```diff
@@ -32,11 +32,15 @@ sealed class JetpackButtonViewHolder(@LayoutRes layout: Int, parent: ViewGroup)
         }
     }
 
-    internal fun Button.updateState(buttonState: ActionButtonState, uiHelpers: UiHelpers) {
+    internal fun MaterialButton.updateState(buttonState: ActionButtonState, uiHelpers: UiHelpers) {
         updateItemViewVisibility(buttonState.isVisible)
         uiHelpers.setTextOrHide(this, buttonState.text)
         isEnabled = buttonState.isEnabled
         setOnClickListener { buttonState.onClick.invoke() }
+        buttonState.iconRes?.let {
+            iconGravity = MaterialButton.ICON_GRAVITY_TEXT_START
+            icon = this.context.getDrawable(it)
```

Comment:
Minor (🔍): Consider removing `this.` as it is not necessary in this context.

Reply:
Removed in 13a5180